drivers/nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -847,17 +864,27 @@ enum op_band {
 	BAND_24G
 };
 
+/**
+ * @brief This enum defines keep alive status.
+ */
+enum nrf_wifi_keep_alive_status {
+	/** Keep alive feature disabled */
+	KEEP_ALIVE_DISABLED = 0,
+	/** Keep alive feature enabled */
+	KEEP_ALIVE_ENABLED = 1
+};
+
 /**
  * @brief This enum specifies the type of frames used to retrieve buffered data
  *  from the AP in power save mode.
  */
-enum data_retrieve_mechanism {
-	/** Retrieves data from the AP using a PS-Poll frame */
-	PS_POLL_FRAME,
-	/** Retrieves data from the AP using a QoS Null frame */
-	QOS_NULL_FRAME,
-	/** For future implementation. The RPU will decide which frame to use */
-	AUTOMATIC
+enum ps_exit_strategy {
```

Comment:
How is this change related to `Offloaded raw TX` feature? Shouldn't this be clubbed with the `PS exit strategy` commit?

Generic comment: Please combine `related` changes together in 1 commit.

Reply:
Committed as a separate commit